admin/registries/index.md

```diff
@@ -37,3 +37,10 @@ When done, click **Import**.
 To delete a registry, go to the **Images** > **Registries** page. Find the
 registry that you'd like to remove, click its **horizontal ellipses** icon, and
 select **Delete**.
+
+## Unsupported registries
+
+The following registries do not behave as indended inside Coder at this time.
+
+- Amazon ECR
```

Comment:
I made a workaround for ECR can be periodically run every 10 hours to update the token in Coder with the API. Not sure if it is appropriate to mention here: https://github.com/bpmct/coder-ecr-sync

Reply:
I think if we're going to call it out as unsupported, we shouldn't link to the workaround. WDYT?

Reply:
Makes sense to me. The workaround is bad too.